src/PimcoreDevkitBundle/Service/DataObjectService.php

```diff
@@ -34,22 +35,16 @@ public function getOrCreateObjectFolder($parentId, $key)
         $key    = DataObject\Service::getValidKey($key, 'object');
         $path   = $parent->getRealFullPath() . '/' . $key;
 
-        $folder = Folder::getByPath($path);
-        if (!$folder instanceof Folder) {
-            $folder = Folder::create([
-                'o_parentId'         => $parentId,
-                'o_creationDate'     => time(),
-                'o_userOwner'        => 0,
-                'o_userModification' => 0,
-                'o_key'              => $key,
-                'o_published'        => true,
-                'o_locked'           => true,
-            ]);
-        }
-
-        return $folder;
+        return DataObject\Service::createFolderByPath($path);
```

Comment:
To nie jest dokładnie ta sama funkcjonalność. Pomija published i locked, nie wiem co z userami. Co sądzisz?